liquibase-core/src/main/java/liquibase/command/CommandArgumentDefinition.java

```diff
@@ -217,6 +243,11 @@ public String toString() {
             return this.defaultValue(defaultValue, description);
         }
 
+        public Building<DataType> interactivePrompt(InteractivePromptableCustomizationWrapper promptableCustomizationWrapper) {
```

Comment:
I like having the promotability defined as part of the CommandArgumentDefinition. But, the InteractivePromptableCustomizationWrapper name seems a bit unweildy

Reply:
Do you have an alternative suggestion? I agree this is unwieldy. How about `CustomizedInteractivePrompt`?